src/operator/mxnet_op.h

```diff
@@ -334,6 +365,24 @@ struct Kernel<OP, cpu> {
     }
 #endif
   }
+
+  template<typename ...Args>
+  inline static void LaunchEx(mshadow::Stream<cpu> *s, const int N, Args... args) {
```

Comment:
?

Reply:
(look familiar)

Reply:
I have same name function in my tuner branch :(

Reply:
launch 2D maybe?

Reply:
Sure. 

Btw, if the actual number of OMP cores that run the for loop differ from the omp_cores value, will the calculation still be correct (albeit a little less optimal)?

Reply:
It should be.